README.md

```diff
@@ -20,24 +20,28 @@ title: Adapted Strength Web Application
 
 ## Team
 
-### Development 
+### Team Leader 
 - Jonathan Camarena Camacho
-    - Discord: kyperbelt
+    - Discord: `kyperbelt`
+### Database developers
 - Danny Zhou
-    - Discord: Timid
+    - Discord: `timid`
+- David Castrejon
+    - Discord: `bartholamewthethird`
+### Backend developers
 - Jeff Huang
     - Discord: `mrb1ue`
-- Ahmed Taeha
-    - Discord: 'ataeha'
+- Blake Danz
+    - Discord: `buh_la_key`
 - Amir Talakoob
-    - Discord: darkscorpion13
-- David Castrejon
-    - Discord: 'bartholamewthethird'
-- Casey Paras
-    - Discord: Lumikha
+    - Discord: `darkscorpion13`
 - Joshua Cupler
-    - Discord: Johncup
-- Blake Danz
-    - Discord: Buh la key
+    - Discord: `johncup`
+### Frontend develoeprs
```

Comment:
developers*

Reply:
Thanks for the check. Issue has be resolved and will be pushed/committed today.